__tests__/app.test.js

```diff
@@ -403,3 +403,30 @@ describe("PATCH: /api/articles/:article_id", () => {
     });
   });
 });
+
+describe("DELETE: /api/comments/:comment_id", () => {
+  describe("DELETE: 202", () => {
```

Comment:
Here i would expect a 204 no content. If this is the case there will be no body to assert against